lib/commands/connect/auth_connection.js

```diff
@@ -34,13 +34,13 @@ module.exports = {
             callbackServer();
 
             try {
-              let args = {'environment':'production', 'next': 'http://127.0.0.1:18000'};
+              var args = {'environment':'production', 'next': 'http://127.0.0.1:18000'};
```

Comment:
This is what immediately caught my attention.   Changing a `let` to a `var` in a block-level try/catch could be very bad if `args` is used anywhere else.